roles/docker.ubuntu/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+---
+# tasks file for docker.ubuntu
+- name: Ensure that Ubuntu version is supported
+  fail:
+    msg: "{{ ansible_distribution_version }} is not an acceptable version of Ubuntu for this role"
+  when: ansible_lsb.id|lower == "ubuntu" and ( ansible_distribution_version|version_compare('14.04', '<') )
+
+- name: Install docker package
```

Comment:
Can anything be done in order to install the latest docker version?
For example, manually setting apt-get (https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04) to fetch the latest version.